nvm.sh

```diff
@@ -403,6 +403,31 @@ nvm_num_version_groups() {
   nvm_echo "${#NVM_NUM_GROUPS}"
 }
 
+nvm_get_upgrade_version() {
+  if [ $# -lt 1 ]; then
+    nvm_echo 'N/A'
+    return 1
+  fi
+
+  local VERSION
+  VERSION="$1"
+
+  local semver_major
+  semver_major="$(nvm_echo \"$base_version\" | command sed -e 's/v\([0-9]\+\)\.\([0-9]\+\).*/\1/')"
+
+  local semver_minor
+  semver_minor="$(nvm_echo \"$base_version\" | command sed -e 's/v\([0-9]\+\)\.\([0-9]\+\).*/\2/')"
+
+  local install_version
+  if [ $semver_major = '0' ]; then
+    install_version="v0.$semver_minor"
+  else
+    install_version="v$semver_major"
+  fi
+
+  nvm_echo "$(nvm_remote_version \"$install_version\")"
```

Comment:
it might be easier to test if you keep the `nvm_remote_version` call outside this function (ie, just echo the install_version)